extensions/wikia/ArticleVideo/scripts/jwplayer-instant.js

```diff
@@ -0,0 +1,72 @@
+(function () {
+	var videoDetails = window.wgFeaturedVideoData;
+	var videoElementId = 'featured-video__player';
+	var videoId = videoDetails.videoId;
+	var playerInstance = jwplayer(videoElementId);
+	window.featureVideoPlayerInstance = playerInstance;
+
+	function handleTabNotActive(willAutoplay) {
+		var hidden,
+			visibilityChange;
+
+		if (typeof document.hidden !== "undefined") {
+			hidden = "hidden";
+			visibilityChange = "visibilitychange";
+		} else if (typeof document.msHidden !== "undefined") {
+			hidden = "msHidden";
+			visibilityChange = "msvisibilitychange";
+		} else if (typeof document.webkitHidden !== "undefined") {
+			hidden = "webkitHidden";
+			visibilityChange = "webkitvisibilitychange";
+		}
+
+		document.addEventListener(visibilityChange, function () {
+			if (!document[hidden] && willAutoplay && playerInstance.getState() !== 'playing') {
+				playerInstance.play(true);
+			}
+		}, false);
+	}
+
+	require([
+		'wikia.cookies',
+		'wikia.geo',
+		'wikia.instantGlobals'
+	], function (cookies, geo, instantGlobals) {
+		var inAutoplayCountries = geo.isProperGeo(instantGlobals.wgArticleVideoAutoplayCountries),
+			inNextVideoAutoplayCountries = geo.isProperGeo(instantGlobals.wgArticleVideoNextVideoAutoplayCountries),
+			autoplayCookieName = 'featuredVideoAutoplay',
+			willAutoplay = cookies.get(autoplayCookieName) !== '0' && inAutoplayCountries;
+
+		playerInstance.setup({
+			file: "//content.jwplatform.com/videos/" + videoId + ".mp4",
+			mediaid: videoId,
+			autostart: willAutoplay && !document.hidden,
+			mute: willAutoplay,
+			image: "//content.jwplatform.com/thumbs/" + videoId + "-640.jpg",
+			related: {
+				file: "https://cdn.jwplayer.com/v2/playlists/Y2RWCKuS?related_media_id=" + videoId,
+				oncomplete: inNextVideoAutoplayCountries ? 'autoplay' : 'show',
+				autoplaytimer: 5
+			},
+			title: videoDetails.title,
+			description: videoDetails.description
+		});
+
+		handleTabNotActive(willAutoplay);
+
+		if (true || inAutoplayCountries) {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Change this condition so that it does not always evaluate to "true"; some subsequent code is never executed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS2583)